ERCS/erc-7893.md

```diff
@@ -0,0 +1,706 @@
+---
+eip: 7893
+title: DeFi Protocol Solvency Proof Mechanism
+description: An interface for verifying and reporting DeFi protocol solvency status through smart contracts
+author: Sean Luis Guada Rodríguez (@SeanLuis) <seanluis47@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/tbd
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-01-30
+requires: 20, 165
+---
+
+## Abstract
+
+A standardized interface that enables DeFi protocols to implement verifiable solvency proofs through smart contracts. The standard defines methods for reporting assets, liabilities, and financial metrics, enabling real-time verification of protocol solvency.
```

Comment:
Your abstract does a good job of describing what your proposal does, but is lacking an explanation of the how. You should include enough detail here to give the reader a high-level (but still technical) overview of how everything works.